ignite/templates/app/files/app/app_config.go.plush

```diff
@@ -3,8 +3,6 @@ package app
 import (
```

Comment:
the import sorting doesn't match golangci-lint rules: https://github.com/ignite/cli/blob/main/.golangci.yml#L46-L53